packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -358,6 +358,11 @@ class Draggable<T extends Object> extends StatefulWidget {
 }
 
 /// Makes its child draggable starting from long press.
+///
+/// See also:
+///
+///  * [Draggable]
+///  * [DragTarget], a widget that receives data when the [LongPressDraggable] is dragged.
```

Comment:
Doesn't it only receive the data when the draggable is dropped on the target?

Reply:
@goderbauer kinda; It provides the following events:

this.onWillAccept -> Receives data
this.onAccept -> Receives data
this.onAcceptWithDetails -> Receives data
this.onLeave -> Doesn't
this.onMove -> Doesn't

But this chain of events are fired on drag.